RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_tools.py

```diff
@@ -0,0 +1,115 @@
+import FWCore.ParameterSet.Config as cms
+
+def calibrateReducedEgamma(process):
+    process.load("RecoEgamma.EgammaTools.calibratedEgammas_cfi")
+    process.calibratedPhotons.produceCalibratedObjs = cms.bool(False)
```

Comment:
for all cases where a parameter is expected to exist already, please do not specify its type on RHS. This is safer against typos

Reply:
done (at least for this file)